docs/language/accounts.mdx

```diff
@@ -167,6 +171,25 @@ to the `prepare` phase of the transaction.
   which discard their changes upon completion. 
   Attempting to use this function outside of a script will cause a type error. 
 
+## Identity
+Identity can only be obtained as a reference from an AuthAccount and has a single `address` field. 
+
+The main use case for Identity is to send it from a transaction into a contract method to know the address of who called it. 
+
+```cadence
+
+// example fun 
+pub fun someMethod(identity: &Identity) {
+   log("I was called by ".concat(identity.address))
+}
+
+transaction() {
+    prepare(signer: AuthAccount) {
+				someMethod(signer.identity)
```

Comment:
Whitespace shenanigans in a few places in this diff. Are they just an artifact of the medium or is the source code going to look weird?